packetbeat/protos/http/http.go

```diff
@@ -415,14 +415,31 @@ func (http *httpPlugin) handleHTTP(
 	}
 }
 
+func (http *httpPlugin) flushResponses(conn *httpConnectionData) {
+	for !conn.responses.empty() {
+		debugf("Response from unknown transaction. Reporting error.")
```

Comment:
If there's any identifying information available about the flow it would likely be beneficial to include a few details in the log. The same in `flushRequests`.

Reply:
Done